http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -328,6 +335,60 @@ class LeoPubsubMessageSubscriber[F[_]: Timer: ContextShift](
       _ <- runtimeConfig.cloudService.process(runtime.id, RuntimeStatus.Starting).compile.drain
     } yield ()
 
+  private[monitor] def handleCreateDiskMessage(msg: CreateDiskMessage, now: Instant)(
+    implicit traceId: ApplicativeAsk[F, AppContext]
+  ): F[Unit] = {
+    val createDisk = for {
+      diskResult <- googleDiskService.createDisk(
+        msg.googleProject,
+        msg.zone,
+        Disk
+          .newBuilder()
```

Comment:
For `createDisk`, don't we have `blockSize`, `diskType` fields too?

Reply:
oh we do, the corresponding fields in google would be `type` and `physicalBlockSizeBytes` I believe. I need to add this to wb-libs as well (it'll be a different `diskClient.insertDisk` method we need to use), should I create a new ticket to add this to wb-libs and then add `blockSize` and `diskType` in the leo create disk call? Alternatively, i can hold off on merging this PR, make the changes in wb-libs and then add the leo changes to this PR.

Reply:
Oh wait, nevermind I think I can just make the changes in leo since we pass a `Disk` object to the google API call

Reply:
Yup, latest change looks good 👍